requirements.txt

```diff
@@ -2,7 +2,6 @@ astropy
 ephem
```

Comment:
I think these requirements should be frozen, is there a reason not to do this? Aside from ipython that I think is used only for debugging purposes I think it's the best way to go